🐛 Bug: Setup script crashes with [ERR_MODULE_NOT_FOUND] for .all-contributorsrc.json

### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/template-typescript-node-package/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

You should be able to run `pnpm run setup`.

### Actual

```plaintext
$ pnpm run setup

> template-typescript-node-package@1.18.0 setup /Users/josh/repos/template-typescript-node-package
> npx --yes zx --quiet script/setup.js

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/josh/repos/template-typescript-node-package/.all-contributorsrc.json' imported from /Users/josh/repos/template-typescript-node-package/script/setup.js
    at new NodeError (node:internal/errors:387:5)
    at finalizeResolution (node:internal/modules/esm/resolve:404:11)
    at moduleResolve (node:internal/modules/esm/resolve:965:10)
    at defaultResolve (node:internal/modules/esm/resolve:1173:11)
    at nextResolve (node:internal/modules/esm/loader:173:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:852:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:439:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

### Additional Info

It's because I renamed the file in #98.

--- a/script/setup.js
+++ b/script/setup.js
@@ -3,10 +3,6 @@ import chalk from "chalk";
 import { promises as fs } from "fs";
 import prettier from "prettier";
 
-import existingContributors from "../.all-contributorsrc.json" assert { type: "json" };
-import existingPackage from "../package.json" assert { type: "json" };
-import outcomeLabels from "./labels.json" assert { type: "json" };
-
 let caughtError;
 
 try {
@@ -87,8 +83,19 @@ try {
 	console.log();
 	console.log(chalk.gray`Hydrating package metadata locally...`);
 
+	async function readFileAsJSON(filePath) {
+		return JSON.parse((await fs.readFile(filePath)).toString());
+	}
+
+	const [existingContributors, existingPackage, outcomeLabels] =
+		await Promise.all([
+			readFileAsJSON("./.all-contributorsrc"),
+			readFileAsJSON("./package.json"),
+			readFileAsJSON("./script/labels.json"),
+		]);
+
 	await fs.writeFile(
-		"./.all-contributorsrc.json",
+		"./.all-contributorsrc",
 		prettier.format(
 			JSON.stringify({
 				...existingContributors,

